### PR TITLE
Auto-push new PHP.net bugs

### PR DESCRIPTION
**Pull request by kelunik**

### CODE REVIEW

**Comment by PeeHaa** [comment]
Considering @JayPHP has been busy with writing tests last week it would be a waste if plugins get added at this point without tests.

Please add unit tests <3

**Comment by kelunik** [comment]
Plugins mostly don't have any tests right now, and if they have, they mostly just test to return the right command endpoint. This plugin doesn't even have one. But I just added a single test at least.

File: src/Plugins/PHPBugs.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<?php  declare(strict_types=1);
+namespace Room11\Jeeves\Plugins;
```
Comment (DaveRandom) [inline src/Plugins/PHPBugs.php]: I realise this is copypasta from another file in Jeeves as there are several like this for some reason but can you unfuck this indentation please?

```php
<?php declare(strict_types=1);

namespace Room11\Jeeves\Plugins;
```

Comment (kelunik) [inline src/Plugins/PHPBugs.php]: Done.

**Pull request merged**